index.md

```diff
@@ -69,6 +69,7 @@ benefit from these resources. You can find posts and discussion on
 
 # Translations
 
+- [Turkish: missing-semester-tr.github.io](https://missing-semester-tr.github.io/)
```

Comment:
Let's keep these alphabetized if you don't mind :)

Reply:
Changed it, Could you check it again?